### ISSUE TITLE
[UX] Segments with long names get cut off in contact profile

### ISSUE THREAD

**Issue by RCheesley** [mautic/mautic#14069]
### Mautic Series

5.1.x series

### Mautic installed version

5.x branch

### Way of installing

I cloned the Mautic repo (not recommended or supported)

### PHP version

8

### What happened?

When checking another PR I noticed that segments with a long name are not being wrapped, and as such are cut off visually on the contact profile:
![screenshot-mautic ddev site-2024 08 20-18_18_48](https://github.com/user-attachments/assets/08075df5-1bc9-471b-95f9-b3aa21fc33c8)

### How can we reproduce this issue?

Step 1: Install sample data in a local/dev environment with the command `ddev exec bin/console d:f:l`
Step 2: Check contact called Lewis Syed and look at the segments list

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by RCheesley** [comment]
cc @mautic/ux-ui-tiger-team

**Comment by LordRembo** [comment]
@RCheesley I think this issue is fixed by another merged PR from @andersonjeccel. I see an ellipsis when trying it out in 5.2
![Screenshot 2024-12-12 at 17 29 30](https://github.com/user-attachments/assets/810ee739-deba-418c-9ba1-1e6cf14966c3)

**Comment by RCheesley** [comment]
AWesome, thanks for this @LordRembo - better than having it disappearing off into the ether!

**Comment by andersonjeccel** [comment]
- [ ] check if I implemented this solution using a Twig template that converts the complete label into a tooltip for things that got cut
